src/app/DownloadsDialog.qml

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2015-2016 Canonical Ltd.
+ *
+ * This file is part of morph-browser.
+ *
+ * morph-browser is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * morph-browser is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.9
+import QtWebEngine 1.9
+import Ubuntu.Components 1.3
+import Ubuntu.Components.Popups 1.3
+import QtQuick.Layouts 1.3
+import Ubuntu.Content 1.3
+import webbrowsercommon.private 0.1
+
+import "MimeTypeMapper.js" as MimeTypeMapper
+import "FileUtils.js" as FileUtils
+
+Popover {
```

Comment:
The background of the "recent downloads" popup is very difficult to see on a light theme. Is this expected?

![image](https://user-images.githubusercontent.com/21966173/99996866-f89cf880-2d81-11eb-80e5-17588fe2f8a6.png)

Reply:
I used the default theme/color for Popover so I guess this is just the result of that. Maybe we can improve this by the changing the background a bit. Not sure yet if we can adjust the shadows of the popover.

Reply:
Popover relays on the dark layer underneath :)